varats/varats/plots/repository_churn.py

```diff
@@ -200,8 +200,15 @@ def draw_code_churn_for_revisions(
     churn_data = build_revisions_churn_table(
         project_name, commit_map, revisions
     )
+
+    # TODO: Fix duplicated revisions on x-axis caused by the prepended
+    #  time_id when the code_churn is activated. Relates to
+    #  https://github.com/se-sic/VaRA-Tool-Suite/pull/453
+
+    # TODO: Move shown revision length to PlotConfig with length=10 as
+    #  default. Relates to https://github.com/se-sic/VaRA-Tool-Suite/pull/453
     revisions = churn_data.time_id.astype(str) + '-' + churn_data.revision.map(
-        lambda x: x[:10]
+        lambda x: x.hash[:10]
```

Comment:
This case is so common that `FullCommitHash` has its own property for this:

```suggestion
        lambda x: x.short_hash
```